prow/oss/terraform/modules/alerts/main.tf

```diff
@@ -12,6 +12,44 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
+resource "google_monitoring_alert_policy" "ci-absent" {
+  for_each     = var.prow_components
+  project      = var.project
+  display_name = "ci-absent-${each.key}"
+  combiner     = "OR" # required
+
+  conditions {
+    display_name = "ci-absent-${each.key}"
+
+    condition_monitoring_query_language {
+      duration = "600s"
+      query    = <<-EOT
+      fetch k8s_container
+      | metric 'kubernetes.io/container/restart_count'
+      | filter
+          (resource.container_name == '${each.key}' && resource.namespace_name == '${each.value.namespace}')
+      | align rate(1m)
+      | every 1m
+      | group_by [], [row_count: row_count()]
```

Comment:
It might be more appropriate to output the sum or restart_counts value (the number of restarts in the 1m alignment period across pods) rather than the row_count (the number of pods).

Reply:
Sorry should have mentioned, this was generated from UI, by simply selecting aggregating by `count` and interval being `1m`, not sure how those selection translated to the `group_by` line here